### ISSUE TITLE
Issue with missing Access-Control-Allow-Origin

### ISSUE THREAD

**Issue by enragedginger** [zerg000000/simple-cors#1]
Hello, thanks for putting this library together. I tried integrating it into a new Clojure + Ring + Compojure web app I'm building and ran into an issue when making CORS calls via a CLJS app. I'm able to reproduce this behavior in both Firefox and Chrome.

On the server, I'm doing this:
```
(cors/wrap {:cors-config {:allowed-request-methods [:get :put :post :delete]
                                :allowed-request-headers ["accept" "accept-encoding" "accept-language" "connection"
                                                          "authorization" "content-length" "cookie" "content-type" "dnt"
                                                          "host" "referer" "user-agent" "origin"]
                                :allow-credentials? true
                                :origins "*"
                                }})
```

The error in the browser console looks like this:
```
Access to XMLHttpRequest at 'http://localhost:9000/login' from origin 'http://localhost:3449' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

Should this middleware set the `Access-Control-Allow-Origin` header on the response or should I be doing that manually?

**Comment by zerg000000** [comment]
Hi @enragedginger ,

Thanks for reporting issue. The issue have been fix with `0.0.8`, could you give it a try?

**Comment by enragedginger** [comment]
Hi @zerg000000,

I decided to just roll my own middleware for this. I looked through the commit history and saw the change you made with forcing header values to strings and I think that should fix the issue I was facing.

Thanks!
